src/Parse.ts

```diff
@@ -151,6 +150,18 @@ const Parse: ParseType = {
   Hooks: undefined,
   Parse: undefined,
 
+  /**
+   * @member {EventuallyQueue} Parse.EventuallyQueue
+   * @static
+   */
+  set EventuallyQueue(queue: EventuallyQueue) {
```

Comment:
@mortenmo Can you add a unit test for this? `npm run test` you should see code coverage drop as this setter/getter isn't tested.